[25/10/2025, 18:59:37] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 18:59:37","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 18:59:43] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 18:59:43","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 18:59:47] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 18:59:47","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:00:05] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:00:05","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:00:10] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:00:10","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:00:34] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:00:34","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:00:44] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:00:44","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:19:59] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:19:59","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:20:42] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:20:42","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:25:04] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 19:25:04","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async MySQLQueryInterface.addIndex (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\abstract\\query-interface.js:250:12)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:999:7)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 19:44:40] INFO: Sesión cerrada {"ip_cliente":"::1"}
[25/10/2025, 19:45:50] ERROR: Credenciales inválidas - Usuario no encontrado {"codigo_error":"400","ip_cliente":"::1","usuario":"20221061@uthh.edu.mx"}
[25/10/2025, 19:46:01] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::1","fecha_hora":"25/10/2025, 19:46:01"}
[25/10/2025, 19:51:27] INFO: Sesión cerrada {"ip_cliente":"::1"}
[25/10/2025, 19:51:43] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::1","fecha_hora":"25/10/2025, 19:51:43"}
[25/10/2025, 22:35:37] INFO: Sesión cerrada {"ip_cliente":"::1"}
[25/10/2025, 22:36:06] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::1","fecha_hora":"25/10/2025, 22:36:06"}
[25/10/2025, 22:53:26] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[25/10/2025, 22:53:26] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[25/10/2025, 23:26:26] ERROR: Credenciales inválidas - Usuario no encontrado {"codigo_error":"400","ip_cliente":"::ffff:127.0.0.1","usuario":"20221061@uthh.edu.mx"}
[25/10/2025, 23:28:35] INFO: Inicio de sesión exitoso {"usuario_id":"4c4d86e0-036c-4b6e-9bac-394ecc0754ef","correo":"20221061@uthh.edu.mx","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"25/10/2025, 23:28:35"}
[25/10/2025, 23:37:38] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:37:38","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:40:08] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:40:08","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:40:29] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:40:29","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:40:49] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:40:49","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:41:44] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:41:44","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:41:58] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:41:58","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:42:12] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:42:12","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:43:58] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:43:58","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[25/10/2025, 23:44:12] CRITICAL: Fallo al iniciar el servidor o conectar a la base de datos {"codigo_error":"SERVER_START_FAIL","fecha_hora":"25/10/2025, 23:44:12","stack":"Error\n    at Query.run (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\dialects\\mysql\\query.js:52:25)\n    at C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:315:28\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Usuario.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\model.js:984:11)\n    at async Sequelize.sync (C:\\Users\\luishdez\\Desktop\\puntoshein\\backend\\node_modules\\sequelize\\lib\\sequelize.js:377:9)\n    at async main (file:///C:/Users/luishdez/Desktop/puntoshein/backend/src/index.js:17:7)","db":"puntoshein","ip_servidor":"localhost"}
[26/10/2025, 1:00:46] WARN: Faltan campos obligatorios {"codigo_error":"400","ip_cliente":"::1"}
[26/10/2025, 1:03:34] WARN: Faltan campos obligatorios {"codigo_error":"400","ip_cliente":"::1"}
[26/10/2025, 1:22:53] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 1:23:12] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 1:23:12"}
[26/10/2025, 1:31:14] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 1:31:30] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 1:31:30"}
[26/10/2025, 1:34:21] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 1:34:41] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 1:34:41"}
[26/10/2025, 1:47:58] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 1:48:14] INFO: Inicio de sesión exitoso {"usuario_id":"4c4d86e0-036c-4b6e-9bac-394ecc0754ef","correo":"20221061@uthh.edu.mx","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 1:48:14"}
[26/10/2025, 2:33:34] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 3:45:53] INFO: Inicio de sesión exitoso {"usuario_id":"4c4d86e0-036c-4b6e-9bac-394ecc0754ef","correo":"20221061@uthh.edu.mx","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 3:45:53"}
[26/10/2025, 3:46:11] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 3:48:47] INFO: Solicitud de recuperación de contraseña {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::ffff:127.0.0.1"}
[26/10/2025, 3:57:40] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::1","fecha_hora":"26/10/2025, 3:57:40"}
[26/10/2025, 4:10:49] ERROR: Credenciales inválidas - Contraseña incorrecta {"codigo_error":"400","ip_cliente":"::ffff:127.0.0.1","usuario":"luis2004hdez@gmail.com"}
[26/10/2025, 4:10:59] INFO: Inicio de sesión exitoso {"usuario_id":"35067a73-07c8-4aef-a781-a63939d84884","correo":"luis2004hdez@gmail.com","ip_cliente":"::ffff:127.0.0.1","fecha_hora":"26/10/2025, 4:10:59"}
[26/10/2025, 4:24:09] INFO: Sesión cerrada {"ip_cliente":"::1"}
[26/10/2025, 4:53:33] INFO: Sesión cerrada {"ip_cliente":"::ffff:127.0.0.1"}
